ai_coach/ai_coach.py

```diff
@@ -115,21 +137,42 @@ def student_view(self, context=None):
         frag = Fragment(html)
         frag.add_css(self.resource_string("static/css/ai_coach.css"))
         frag.add_javascript(self.resource_string("static/js/src/ai_coach.js"))
-        frag.initialize_js('AICoachXBlock')
+        frag.initialize_js('AICoachXBlock', json_args=self.get_context())
         return frag
 
+    def get_chat_completion(
+            self, prompt='', model='gpt-3.5-turbo', temperature=0.5, max_tokens=150, n=1
+    ):
+        """ Returns the improvement for student answer using ChatGPT Model """
+
+        messages = [{"role": "user", "content": prompt}]
+        try:
+            response = openai.ChatCompletion.create(
+                    messages=messages,
+                    model=model,
+                    temperature=temperature,
+                    max_tokens=max_tokens,
+                    n=n
+            )
+        except Exception as err:
+            log.error(err)
+            return {'error': 'Unable to connect to AI-coach. Please contact your administrator'}
```

Comment:
We should mark this and rest of the strings for translations same approach as in other xblocks like h5pxblock